api/src/opentrons/hardware_control/api.py

```diff
@@ -962,16 +962,27 @@ async def dispense(
         else:
             dispense_spec.instr.remove_current_volume(dispense_spec.volume)
 
-    async def blow_out(self, mount: top_types.Mount) -> None:
+    async def blow_out(
+        self, mount: top_types.Mount, microliters: Optional[float] = None
```

Comment:
```suggestion
        self, mount: top_types.Mount, volume: Optional[float] = None
```

Keeps it consistent with other functions that take volumes